lib/ansible/module_utils/basic.py

```diff
@@ -1630,32 +1614,7 @@ def _check_locale(self):
                            to_native(e), exception=traceback.format_exc())
 
     def _handle_aliases(self, spec=None, param=None):
-        # this uses exceptions as it happens before we can safely call fail_json
```

Comment:
This comment might need to stay in the stub function.  At least until we move fail_json() and exit_json() out.